packages/app/src/components/Bookmarks/BookmarkFolderItem.tsx

```diff
@@ -38,7 +38,7 @@ const BookmarkFolderItem: FC<BookmarkFolderItemProps> = (props: BookmarkFolderIt
   const [isCreateAction, setIsCreateAction] = useState<boolean>(false);
   const [isDeleteFolderModalShown, setIsDeleteFolderModalShown] = useState<boolean>(false);
 
-  const childCount = useCallback((): number => {
+  const getChildCount = useCallback((): number => {
```

Comment:
@yuki-takei 
これって useCallback でもいけるが、 中身の処理的にuseMemoを使った方がいいのでしょうか。

```suggestion
  const childCount = useMemo((): number => {
```

Reply:
CountBadge にはメソッドの参照渡しではなく結果を渡しているだけなので、パフォーマンス的にはどちらも同程度かと。
可読性の面で useMemo でいいとは思う。

Reply:
@mudana-grune 

Please use useMemo 🙏 

```suggestion
  const childCount = useMemo((): number => {
```